openrtb_ext/publisher_test.go

```diff
@@ -0,0 +1,60 @@
+package openrtb_ext
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
Do we really need these tests? We're basically testing json library's `Unmarshal` method here which isn't needed. We can trust the json library

Reply:
Mansi, I can remove the `openrtb_ext/publisher_test.go` test case file if you want, but similar test case files that pretty much only test the `json.Unmarshal()` function also exist in the `openrtb_ext/` directory such as `openrtb_ext/request_test.go` and `openrtb_ext/site_test.go`. I noticed those two when I first coded this PR and figured I had to create similar tests to assert the proper unmarshaling of the `ExtPublisherPrebid` struct that this PR implements. Please let me know if you want me to get rid of `openrtb_ext/publisher_test.go`.

Reply:
Yeah, I don't think we need to test the `json.Unmarshal` method as part of our code so if you don't mind removing it then that will be great. Thanks!

Reply:
Done